merino/web/api_v1.py

```diff
@@ -114,6 +118,25 @@ async def suggest(
     return JSONResponse(content=jsonable_encoder(response))
 
 
+def emit_suggestions_per_metrics(
+    metrics_client: Client, suggestions: list, active_providers: list
```

Comment:
Nit: we could avoid the `list()` call above w/ `Iterable`:

```suggestion
    metrics_client: Client, suggestions: list, active_providers: Iterable[str]
```

`suggestions` could also type to `list[BaseSuggestion]` but I'm not sure how strict we care to be with such arguments

Reply:
This is much nicer, thanks! 

> suggestions could also type to list[BaseSuggestion] but I'm not sure how strict we care to be with such arguments

I think being explicit with `BaseSuggestion` is a good idea because the `emit_suggestions_per_metrics` function introspects into the Suggestion object. So I'll also add this change!